packages/react-components/react-combobox/src/components/Combobox/Combobox.types.ts

```diff
@@ -9,6 +9,7 @@ import type {
   ComboboxBaseState,
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31151&commitId=40c5d55653d2be66b45190523b42989b4dd63552&env=prod&clientType=FLUENTUI)


  <details><summary><b>Combobox Converged</b> 22 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Combobox Converged.Open with inlinePopup.active option.chromium.png|23| Changed|
|Combobox Converged.Disabled option.hover.chromium.png|23| Changed|
|Combobox Converged.Open with grouped options.hover.chromium.png|58| Changed|
|Combobox Converged.Open with inlinePopup.hover.chromium.png|23| Changed|
|Combobox Converged.Disabled option.active option.chromium.png|23| Changed|
|Combobox Converged.Open with inlinePopup.default.chromium.png|32| Changed|
|Combobox Converged.Open.default.chromium.png|32| Changed|
|Combobox Converged.When rendering inline, it should render on top of relatively positioned elements.default.chromium.png|32| Changed|
|Combobox Converged.Option with long content.hover.chromium.png|22| Changed|
|Combobox Converged.Disabled option.default.chromium.png|23| Changed|
|Combobox Converged.When rendering inline, it should render on top of relatively positioned elements.hover.chromium.png|23| Changed|
|Combobox Converged.Open with grouped options.active option.chromium.png|58| Changed|
|Combobox Converged.With selection.default.chromium.png|35| Changed|
|Combobox Converged.Open.active option.chromium.png|23| Changed|
|Combobox Converged.Option with long content.default.chromium.png|373| Changed|
|Combobox Converged.Open with grouped options.default.chromium.png|68| Changed|
|Combobox Converged.With selection.hover.chromium.png|23| Changed|
|Combobox Converged.Option with long content.active option.chromium.png|22| Changed|
|Combobox Converged.With selection.active option.chromium.png|23| Changed|
|Combobox Converged.Open.hover.chromium.png|23| Changed|
|Combobox Converged.With multiselect selection.default.chromium.png|32| Changed|
|Combobox Converged.When rendering inline, it should render on top of relatively positioned elements.active option.chromium.png|23| Changed|

</details>


  <details><summary><b>Dialog</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dialog.Integration Combobox Inline.chromium.png|57| Changed|

</details>


  <details><summary><b>Dropdown Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dropdown Converged.When rendering inline, it should render on top of relatively positioned elements.default.chromium.png|10| Changed|

</details>


  <details><summary><b>TimePicker compat</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TimePicker compat.Default.default.chromium.png|74| Changed|
|TimePicker compat.With 12 Hour Format.default.chromium.png|69| Changed|
|TimePicker compat.Show Seconds.default.chromium.png|97| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>